src/main/java/com/czertainly/core/service/acme/impl/AcmeServiceImpl.java

```diff
@@ -543,7 +543,7 @@ public void finalizeOrder(AcmeOrder order, AcmeJwsRequest jwsRequest, boolean is
         logger.debug("Initiating issue Certificate for Order with ID: {}", order.getOrderId());
         ClientCertificateSignRequestDto certificateSignRequestDto = new ClientCertificateSignRequestDto();
         certificateSignRequestDto.setAttributes(getClientOperationAttributes(false, order.getAcmeAccount(), isRaProfileBased));
-        certificateSignRequestDto.setPkcs10(decodedCsr);
+        certificateSignRequestDto.setRequest(decodedCsr);
```

Comment:
I do not think the default `format` will be assigned here because we are not using builder as with other DTOs. It should be set to `pkcs10` in case of ACME, ACME does not work with `crmf`.